setup.cfg

```diff
@@ -7,21 +7,29 @@ long_description_content_type = text/markdown
 url = https://github.com/lapix-ufsc/lapixdl
 author = LAPiX
 license = MIT
-license_file = LICENSE
+license_file = LICENCE
+classifiers =
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: Implementation :: CPython
 
 [options]
 packages = find:
 install_requires =
+    matplotlib
     numpy
     pandas
-    tqdm
     seaborn
-    pandas
-    matplotlib
-
+    tqdm
 python_requires = >=3.6
 
 [options.packages.find]
 exclude =
     tests*
-    data*
```

Comment:
Why did you include `data`?

Reply:
I just forgot to delete, can delete this

Reply:
I got it wrong. You already deleted it, it is OK.